service/test/integration_test.go

```diff
@@ -1070,3 +1070,12 @@ func TestSkipPrologue2(t *testing.T) {
 		}
 	})
 }
+
+func TestTypesCommand(t *testing.T) {
+	withTestClient("testvariables2", t, func(c service.Client) {
+		state := <-c.Continue()
+		assertNoError(state.Err, t, "Continue()")
+		_, err := c.ListTypes("")
+		assertNoError(err, t, "ListTypes()")
+	})
```

Comment:
Let's assert about the filter working correctly here, possibly by filtering to a custom type and asserting only it comes back.